lib/ls-js-sdk.js

```diff
@@ -9,9 +8,9 @@ var glucose = require("ls-glucose");
 /**
   * (Public) Function that initialises the SDK setting the configuration object
   * @param config {Object} Configuration object, which accepts following properties:
-  *   app_id {String} An API ID (required)
-  *   app_key {String} An API Key (required)
-  *   api_user {String} User ID (optional)
+  *   app_token {String} An authentication token
+  *   app_id {String} An API ID (required) DEPRECATED
```

Comment:
I am not sure we would say app_id and app_key are actually deprecated, I rather see tokens and app_id/app_key as two alternative means of authentication. Some internally used web app (not accessible to the world) may still want to use app_id/app_key for auth. It would be interesting to know @webpolk 's view.